tools/devops/device-tests/templates/device-tests.yml

```diff
@@ -279,7 +279,6 @@ steps:
     fi
   env:
     WORKING_DIR: $(System.DefaultWorkingDirectory) 
-    XAMARIN_STORAGE_PATH: $(XAMARIN_STORAGE_PATH)
     TESTS_EXTRA_ARGUMENTS: ${{ parameters.testsLabels }}
     USE_XAMARIN_STORAGE: '${{ parameters.useXamarinStorage }}'
```

Comment:
Why does this one have single quotes when TESTS_EXTRA_ARGUMENTS does not?

Reply:
No diff AFAIK, I'll remove the '